website/content/installation/_index.md

```diff
@@ -145,6 +155,38 @@ configInline:
      - 198.51.100.0/24
 ```
 
+{{% notice note %}}
+If you want to deploy metallb using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), the following value must be set:
+
+```yaml
+speaker:
+  frr:
+    enabled:true
+```
+
+{{% /notice %}}
+
+## Using the MetalLB Operator
+
+The MetalLB Operator is available on OperatorHub. It allows configuring MetalLB via CRDs (although [it's planned to have the configuration being handled directly by MetalLB](https://github.com/metallb/metallb/pull/942)).
+
+{{% notice note %}}
+If you want to deploy metallb using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), you must edit the ClusterServiceVersion resource
+named `metallb-operator`:
+
+```bash
+kubectl edit csv metallb-operator
+```
+
+and change the `BGP_TYPE` environment variable of the `manager` container to `frr`:
+
+```yaml
+- name: METALLB_BGP_TYPE
+  value: frr
+```
+
+The MetalLB Operator allows to configure MetalLB via CRDs.
```

Comment:
This line repeats what has been already said in the first sentence of this section:
"The MetalLB Operator is available on OperatorHub. It allows configuring MetalLB via CRDs"